baselines/disable-packing-test/test/gapic_echo_v1beta1.ts.baseline

```diff
@@ -412,6 +412,39 @@ describe('v1beta1.EchoClient', () => {
             assert((client.innerApiCalls.wait as SinonStub)
                 .getCall(0).calledWith(request, expectedOptions, undefined));
         });
+
+        it('invokes checkWaitProgress without error', async () => {
+            const client = new echoModule.v1beta1.EchoClient({
+                credentials: {client_email: 'bogus', private_key: 'bogus'},
+                projectId: 'bogus',
+            });
+            client.initialize();
```

Comment:
Just to double check, is this sync or async?  For some reason I thought it was async.

Reply:
It returns promise, yes, but we don't want to await for this, coz this will break the callback logic in the generated function calls. If we await for this, then it's sure that the calls are initialized.

Reply:
To add to this - it's `async` but we are not in `async` function and cannot `await` for this. But yes, it runs in background, and all requests will internally wait for its completion.